### PR TITLE
LPS-119966

### PR DESCRIPTION
**Pull request by RitaSilveira**
Jira Ticket :
LPS-137721
LPS-119966

Motivation:
It is needed that documents and media are included so that they can also be listed and found

### CODE REVIEW

**Comment by RitaSilveira** [comment]
https://issues.liferay.com/browse/LPS-137721, cc: @CarlosBrichete

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase [hunk 1]
```diff
@@ -73,6 +73,92 @@ definition {
...
+			WebContent.selectCategory(
+				categoryName = "Category 1",
+				vocabularyName = "Vocabulary Name");
+
+			WebContent.selectCategory(
+				categoryName = "Category 2",
+				vocabularyName = "Vocabulary Name");
+
+			PortletEntry.publish();
+		}
```
Comment (CarlosBrichete) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase:115]: You can add a web content with multiple categories:

WebContent.addWithMultipleCategories(
			categoryNameList = "Category Name 1,Category Name 2,Category Name 3,Category Name 4,Category Name 5",
			vocabularyName = "Vocabulary Name",
			webContentContent = "WC Content Body",
			webContentTitle = "WC Content Title");

**Comment by CarlosBrichete** [comment]
Hey @RitaSilveira , the story is about **Documents**, not web content. In the test you are creating and filtering a web content, which makes no sense. Please resend, thank you.

**Comment by RitaSilveira** [comment]
fixed

**Comment by CarlosBrichete** [comment]
Hey @RitaSilveira , this PR is closed, if you want to send it again please send a new PR, thanks.

**Pull request closed without merging**